flash/image/face_detection/model.py

```diff
@@ -0,0 +1,183 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Callable, Dict, List, Mapping, Optional, Sequence, Type, Union
+
+import pytorch_lightning as pl
+import torch
+from torch import nn
+from torch.optim import Optimizer
+
+from flash.core.data.data_source import DefaultDataKeys
+from flash.core.data.process import Preprocess, Serializer
+from flash.core.finetuning import FlashBaseFinetuning
+from flash.core.model import Task
+from flash.core.utilities.imports import _FASTFACE_AVAILABLE
+from flash.image.face_detection.backbones import FACE_DETECTION_BACKBONES
+from flash.image.face_detection.data import FaceDetectionPreprocess
+
+if _FASTFACE_AVAILABLE:
+    import fastface as ff
+
+
+class FaceDetectionFineTuning(FlashBaseFinetuning):
+    def __init__(self, train_bn: bool = True) -> None:
+        super().__init__(train_bn=train_bn)
+
+    def freeze_before_training(self, pl_module: pl.LightningModule) -> None:
+        self.freeze(modules=pl_module.model.backbone, train_bn=self.train_bn)
+
+
+class DetectionLabels(Serializer):
+    """A :class:`.Serializer` which extracts predictions from sample dict."""
+
+    def serialize(self, sample: Any) -> Dict[str, Any]:
+        sample = sample[DefaultDataKeys.PREDS] if isinstance(sample, Dict) else sample
+        return sample
```

Comment:
```suggestion
        return sample[DefaultDataKeys.PREDS] if isinstance(sample, Dict) else sample
```

small nit but can we collapse?